server/GameController.js

```diff
@@ -225,14 +220,23 @@ module.exports = function(data) {
 			//Update this players hand with a new card, as they have just played one
 			updateHand(playerId, answerText);
 
-			//check if everyone submitted and sends back all the currently submitted answers
 			var allChoicesSubmitted;
 
+			//check if everyone submitted and sends back all the currently submitted answers
 			if (currentRound.answers.length === getNumOfConnectedPlayers()) {
 				// change gametsate to the next stage
+
+				//allow everyone to vote again
+				setAllPlayersAbleToSubmit();
```

Comment:
Don't need this as the function setAllPlayersAbleToSubmit() is also called on page load -> aka when timer is started